### PR TITLE
Added interface for commands which allow the bus to return as soon as…

### PR DESCRIPTION
**Pull request by anti43**
… a value is present (contrary to commands which require a new value to appear on the bus before return). Implement for speed command only yet, I need to verify which commands support it. Docs are very rare. But the effect is massive speedup (200ms wait delay can be reduced to 80), so its possible to have like real-time speed gauges etc

### CODE REVIEW

File: src/main/java/com/github/pires/obd/commands/fuel/FuelTrimObdCommand.java [hunk 1]
```diff
@@ -1,8 +1,7 @@
 /**
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not
  * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *     http://www.apache.org/licenses/LICENSE-2.0
+ * the License at http://www.apache.org/licenses/LICENSE-2.0
```
Comment (pires) [inline src/main/java/com/github/pires/obd/commands/fuel/FuelTrimObdCommand.java]: Can you please remove license formatting?

File: src/main/java/com/github/pires/obd/commands/engine/AbsoluteLoadObdCommand.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2015 Andreas.
```
Comment (pires) [inline src/main/java/com/github/pires/obd/commands/engine/AbsoluteLoadObdCommand.java]: This doesn't follow current copyright notice.

File: src/main/java/com/github/pires/obd/commands/ReturnAsapCommand.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2015 Andreas.
```
Comment (pires) [inline src/main/java/com/github/pires/obd/commands/ReturnAsapCommand.java]: This doesn't follow current copyright notice.

File: src/main/java/com/github/pires/obd/commands/ObdCommand.java [hunk 4]
```diff
@@ -1,8 +1,7 @@
 /**
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not
  * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *     http://www.apache.org/licenses/LICENSE-2.0
+ * the License at http://www.apache.org/licenses/LICENSE-2.0
```
Comment (pires) [inline src/main/java/com/github/pires/obd/commands/ObdCommand.java]: Licensing formatting again.

Comment (anti43) [inline src/main/java/com/github/pires/obd/commands/ObdCommand.java]: hm probbaly thats the IDE's work, I usually do not touch the licensing info
:) I check whats happening

2015-07-16 21:55 GMT+02:00 Paulo Pires <notifications@github.com>:

Comment (anti43) [inline src/main/java/com/github/pires/obd/commands/ObdCommand.java]: fixed

2015-07-16 22:33 GMT+02:00 Andreas Weber <senior.weber@gmail.com>:

**Comment by pires** [comment]
I will merge this now, but next time try to squash your commits for less messier history. Thank you :)

**Pull request merged**